utils/fix_rules.py

```diff
@@ -272,6 +272,25 @@ def rewrite_value_int_str(line):
     return key + ": " + str_value
 
 
+def rewrite_keyless_section(file_contents, yaml_contents, section, content):
+    # For a given section, rewrite the keys in int_keys to be strings. Refuses to
+    # operate if the given section appears more than once in the file. Assumes all
+    # instances of key are an integer; all will get updated.
+    new_contents = file_contents[:]
+
+    sec_ranges = find_section_lines(file_contents, section)
+    if len(sec_ranges) != 1:
+        raise RuntimeError("Refusing to fix file: %s -- could not find one section: %d"
+                           % (path, sec_ranges))
+
+    if len(sec_ranges[0]) != 2:
+        raise RuntimeError("Section has more than one line")
+
+    new_contents[sec_ranges[0][0]] = f"{section}: {content}"
```

Comment:
Are f-strings allowed? Are we expecting utilities to be py2/py3 compatible?

Reply:
I would allow it - even RHEL7 has Python3.6, moreover utilities are not needed for the build.